discovery_engine_core/core/src/config.rs

```diff
@@ -124,11 +128,6 @@ pub struct EndpointConfig {
     /// The maximum age of a news article, in days, after which we no longer
     /// want to display them.
     pub max_article_age_days: usize,
-    /// The timeout after which a provider aborts a request.
-    #[serde(with = "serde_duration_as_milliseconds")]
-    pub timeout: Duration,
-    /// The number of retries in case of a timeout.
-    pub retry: usize,
```

Comment:
why we're removing these options?

Reply:
they are just moved to the provider config. the procedure to configure them from the app side via the JSON `de_config` remains the same.